assets/services/parachute/test_parachute.py

```diff
@@ -0,0 +1,411 @@
+        )
+
+    def tearDown(self):
+        # Re-enable the parachute status stream at default rate
+        encode_and_send_mavlink_command_long(
+            self.parachute_connection,
+            mavutil.mavlink.MAV_CMD_SET_MESSAGE_INTERVAL,
+            mavutil.mavlink.MAVLINK_MSG_ID_PARACHUTE_STATUS,
+            0,
+        )
+        assert_mavlink_command_ack(
+            self, mavutil.mavlink.MAV_CMD_SET_MESSAGE_INTERVAL, mavutil.mavlink.MAV_RESULT_ACCEPTED
+        )
+        msg = self.parachute_connection.udpin.recv_match(
+            type="PARACHUTE_STATUS", blocking=True, timeout=self.status_timeout
+        )
+        msg_2 = self.parachute_connection.udpin.recv_match(
+            type="PARACHUTE_STATUS", blocking=True, timeout=self.status_timeout
+        )
+        self.assertIsNotNone(msg, "Did not recieve a PARACHUTE_STATUS message")
+        self.assertIsNotNone(msg_2, "Did not recieve a second PARACHUTE_STATUS message")
+
+    def test_parachute_status_request(self):
+        # Request one parachute status message
+        encode_and_send_mavlink_command_long(
+            self.parachute_connection,
+            mavutil.mavlink.MAV_CMD_REQUEST_MESSAGE,
+            mavutil.mavlink.MAVLINK_MSG_ID_PARACHUTE_STATUS,
+        )
+        assert_mavlink_command_ack(
+            self, mavutil.mavlink.MAV_CMD_REQUEST_MESSAGE, mavutil.mavlink.MAV_RESULT_ACCEPTED
+        )
+        msg = self.parachute_connection.udpin.recv_match(
+            type="PARACHUTE_STATUS", blocking=True, timeout=self.status_timeout
+        )
+        self.assertIsNotNone(msg, "Did not recieve a PARACHUTE_STATUS message")
+
+
+class MavlinkComponentInformationBasicTestCase(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.parachute_connection = MavlinkUDPConnection()
+        cls.msg_timeout = 5
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.parachute_connection.close()
+
+    def setUp(self):
+        msg = self.parachute_connection.udpin.recv_match(
+            type="COMPONENT_INFORMATION_BASIC", blocking=True, timeout=self.msg_timeout
+        )
+        self.assertIsNone(
+            msg, "Recieved a COMPONENT_INFORMATION_BASIC message when the stream should be disabled"
+        )
+
+    def test_component_information_basic_request(self):
+        # Request component information basic message
+        encode_and_send_mavlink_command_long(
+            self.parachute_connection,
+            mavutil.mavlink.MAV_CMD_REQUEST_MESSAGE,
+            mavutil.mavlink.MAVLINK_MSG_ID_COMPONENT_INFORMATION_BASIC,
+        )
+        assert_mavlink_command_ack(
+            self, mavutil.mavlink.MAV_CMD_REQUEST_MESSAGE, mavutil.mavlink.MAV_RESULT_ACCEPTED
+        )
+        msg = self.parachute_connection.udpin.recv_match(
+            type="COMPONENT_INFORMATION_BASIC", blocking=True, timeout=self.msg_timeout
+        )
+        self.assertIsNotNone(msg, "Did not recieve a COMPONENT_INFORMATION_BASIC message")
```

Comment:
since this is only a basic testsuite and `COMPONENT_INFORMATION_BASIC` is shared across many things, I'm not too sure what are some good checks for the content of the message